client/my-sites/plan-features/index.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import PlanFeaturesHeader from './header';
+import PlanFeaturesItemList from './list';
+import PlanFeaturesItem from './item';
+import PlanFeaturesFooter from './footer';
+import { getCurrentPlan } from 'state/sites/plans/selectors';
+import { getPlanPriceObject, getPlan } from 'state/plans/selectors';
+import sitesFactory from 'lib/sites-list';
+import { plansList, getPlanFeaturesObject, PLAN_PREMIUM } from 'lib/plans/constants';
+
+class PlanFeatures extends Component {
+	render() {
+		const {
+			planName,
+			planPrice,
+			popular,
+			current,
+			planConstantObj,
+			features,
+			billingTimeFrame
+		} = this.props;
+
+		const classes = classNames( 'plan-features', {
+			'is-popular': popular
+		} );
+
+		return (
+			<div className={ classes } >
+				<PlanFeaturesHeader
+					current={ current }
+					title={ plansList[ planName ].getTitle() }
+					planType={ planName }
+					price={ planPrice }
+					billingTimeFrame={ billingTimeFrame }
+				/>
+				<PlanFeaturesItemList>
+					{
+						features.map( ( feature, index ) =>
+							<PlanFeaturesItem key={ index }>{ feature.getTitle() }</PlanFeaturesItem>
+						)
+					}
+				</PlanFeaturesItemList>
+				<PlanFeaturesFooter current={ current } description={ planConstantObj.getDescription() } />
+			</div>
+		);
+	}
+}
+
+export default connect( ( state, ownProps ) => {
+	const planProductId = plansList[ ownProps.plan ].getProductId();
+	let sitePlanObj = getCurrentPlan( state, sitesFactory().getSelectedSite() );
```

Comment:
If the selected site is populated, we may be able to avoid needing to populate the sites/plans endpoint that getCurrentPlan depends on.

The site object has a plan property which looks like:
![screen shot 2016-06-15 at 4 26 36 pm](https://cloud.githubusercontent.com/assets/1270189/16100829/f6715d96-3315-11e6-9ba0-c5e5dfe2b10c.png)

We could add a new selector like `getSitePlan` in `sites/selectors`